apps/crawler-fetcher/src/python/crawler_fetcher/handlers/feed_podcast.py

```diff
@@ -112,8 +112,25 @@ def _get_feed_url_from_google_podcasts_url(url: str) -> str:
 
     html = res.decoded_content()
 
-    # <div jsname="<...>" jscontroller="<...>" jsaction="<...>" data-feed="<...>">
-    match = re.search(r'data-feed="(https?://.+?)"', html, flags=re.IGNORECASE)
+    # check whether this is an individual episode URL rather than the show's Google Podcasts homepage; the feed URL
+    # doesn't appear on individual episode pages, so we need to spider to the show's Google Podcasts homepage to get it
+
+    if re.search(r'a class="ik7nMd" jsname="w58EXe"', html, flags=re.IGNORECASE):
```

Comment:
First of all - thanks, very nice catch! Fun trick on determining the feed's URL from episode's URL too.

My suspicion is that `class="ik7nMd"` and `jsname="w58EXe"` get autogenerated on every code change on their end, so whenever they push out a new version, that class name will get updated and this code will break.

I've checked a few feed pages real quick, and every podcast (not episode) page seems to have exactly three `<c-data>` elements, one of which has our feed URL surrounded by some cruft. Could you perhaps parse for that instead?